src/test/java/Homework21.java

```diff
@@ -0,0 +1,21 @@
+import org.testng.annotations.Test;
+
+
+//import static java.lang.Thread.*;
+
+
+public class Homework21 extends BaseTest {
+    @Test
+
+    public void renamePlaylist()  {
+        // openUrl();
+        enterEmail("Iparzyan@gmail.com");
+        enterPassword("MkoIkaMarkMia-789");
+        clickSubmit();
+        doubleclickPlaylistName();
+        enterNewPlaylistName();
+
```

Comment:
**Use in test methods, the following logical split:   GIVEN – WHEN - THEN**
    
For example

```
 @Test

    public void testChangeProfileName (){
        // GIVEN

        String newName  = getRandomString();

        String currentPassword = "te$t$tudent";

        provideEmail("demo@class.com");

        providePassword();

        clickSubmitBtn();

 

        // WHEN

        openUserProfilePage();

        setName(newName);

        setCurrentPassword(currentPassword);

 

 

        // THEN

        Assert.assertTrue(getSuccessPopUp().isDisplayed());

}
```

Please, add Assert

**One test - One Assert**

Reply:
Hello. Thank you for your answer In the LMS homework 21 section Assert is not in requirements .That's why I didn't add it. But I got it.Thank You